lib/elasticity/instance_type_config.rb

```diff
@@ -0,0 +1,30 @@
+module Elasticity
+
+  class InstanceTypeConfig
+
+    attr_accessor :bid_price
+    attr_accessor :type
+    attr_accessor :weighted_capacity
+
+    def initialize
+      @bid_price = nil
+      @type = 'm1.small'
+    end
+
+    def ebs=(ebs_opts = {})
+      @ebs = Elasticity::EBSConfiguration.from_opts(ebs_opts)
+    end
+
+    def to_aws_instance_config
+      {
+        :instance_type => @type,
```

Comment:
So then why is there not no default on @type and a require check here?

Reply:
There is a default; check the constructor. I'm not sure exactly what you mean by "a require check."